helpers/deploymentConfig.ts

```diff
@@ -143,6 +148,12 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
           symbol: "FDUSD",
           decimals: 18,
         },
+        {
+          isMock: true,
```

Comment:
I think we can reuse the MockTWT token deployed in the Isolated pool repo: `0xb99c6b26fdf3678c6e2aff8466e3625a0e7182f8`. wdyt?